### PR TITLE
PRs: 5791, 5755, 5793, 5850, 5857

### PR DESCRIPTION
**Pull request by peiying2**
#5755: MOSS update, hi-IN only
#5791: New, Privacy FAQ page; update Index page
#5793: Add form field strings to newsletter page
#5850 (replaces 5777): Privacy landing page update
#5857: Privacy update on Firefox channel pages

### CODE REVIEW

**Comment by flodolo** [comment]
What about 5793?

**Comment by flodolo** [comment]
5777 seems safe for strings extraction, but I would double check.
5713 is tagged as WIP. Did you check with them if it's ready for string extraction?

**Comment by flodolo** [comment]
Please let me know when you think this is ready for review.

**Comment by peiying2** [comment]
Flod, I want to remove the commit for PR5713 as it may not be final. The rest are ready for review.

**Comment by flodolo** [comment]
> Flod, I want to remove the commit for PR5713 as it may not be final.

If you could do it, it would make things easier on my side to review (i.e. know what really belongs to the PR).

**Comment by flodolo** [comment]
BTW, there's one more
https://github.com/mozilla/bedrock/pull/5857

File: en-US/privacy/index.lang [hunk 1]
```diff
@@ -17,14 +18,33 @@ If you want to make a correction to your information, or you have any questions
...
+Mozilla's Data Privacy Principles inspire our practices that respect and protect people who use the Internet. Learn how these principles shape Firefox and all of our products in this <a href="%(faq)s">FAQ</a>.
+
+
+# This string is obsolete. Do not remove.
 ;Mozilla is an open source project with a mission to improve your Internet experience. This is a driving force behind our data privacy practices. <a href="%(link)s">Read More</a>
 Mozilla is an open source project with a mission to improve your Internet experience. This is a driving force behind our data privacy practices. <a href="%(link)s">Read More</a>
 
 
+## TAG: updates-052018
 ;Our Privacy Notices
```
Comment (flodolo) [inline en-US/privacy/index.lang]: This is not correct, it's the other way around
https://github.com/mozilla/bedrock/pull/5850/files#diff-a8503aba570309aa9167f580d02641edL80

File: en-US/privacy/index.lang [hunk 2]
```diff
@@ -37,19 +57,36 @@ Mozilla Websites, Communications &amp; Cookies
...
+Transparency Report
 
 
+## TAG: updates-052018
+;As an open source project, transparency and openness are an essential part of Mozilla’s founding principles. Our codebases are open and auditable. Our development work is open. Our bi-annual <a href="%(report)s">Transparency Report</a> also demonstrates our commitment to these principles.
+As an open source project, transparency and openness are an essential part of Mozilla’s founding principles. Our codebases are open and auditable. Our development work is open. Our bi-annual <a href="%(report)s">Transparency Report</a> also demonstrates our commitment to these principles.
 
+
+## TAG: updates-052018
+;To review and comment on proposed changes to our privacy policies <a href="%(group)s"> subscribe to Mozilla’s governance group</a>.
```
Comment (flodolo) [inline en-US/privacy/index.lang]: This is not the string in the PR. As said, strings should be extracted via script, not manually.

File: en-US/firefox/channel/index.lang [hunk 3]
```diff
@@ -161,6 +174,12 @@ Get a sneak peek at our next generation web browser, and help us make it the bes
 Test brand new features daily (or… nightly). Enjoy at your own risk.
 
 
+## TAG: privacy_201806
+;Nightly is an unstable testing and development platform. By default, Nightly sends data to Mozilla — and sometimes our partners — to help us handle problems and try ideas. <a href="{%(link)s">Learn what is shared</a>.
+Nightly is an unstable testing and development platform. By default, Nightly sends data to Mozilla — and sometimes our partners — to help us handle problems and try ideas. <a href="{%(link)s">Learn what is shared</a>.
```
Comment (flodolo) [inline en-US/firefox/channel/index.lang]: Link is incorrect (again, don't extract strings by hand, please).

File: en-US/privacy/index.lang (hunk 1 above)
Comment (peiying2) [reply en-US/privacy/index.lang]: Oops, gosh how did I manage to do that.

File: en-US/privacy/index.lang (hunk 2 above)
Comment (peiying2) [reply en-US/privacy/index.lang]: I am really confused with the changes in this PR. When I extract the strings, I get the upper case Governance Group. But the tagged strings suggest lower case.

Comment (flodolo) [reply en-US/privacy/index.lang]: Besides a different capitalization for Governance Group, there's a comma before the link in the string behind `updates-052018`
https://github.com/mozilla/bedrock/pull/5850/files#diff-a8503aba570309aa9167f580d02641edR137

**Pull request merged**